app/enterprise/changelog.md

```diff
@@ -479,6 +493,31 @@ no_version: true
   with command not found". [#7523](https://github.com/Kong/kong/pull/7523)
 
 
+## 2.4.1.2
+**Release Date** 2021/08/12
+
+### Fixes
+
+#### Enterprise
+- Users can now successfully delete workspaces after deleting all entities associated with that workspace.
+  Previously, Kong Gateway was not correctly cleaning up parent-child relationships. For example, creating
+  an Admin also creates a Consumer and RBAC user. When deleting the Admin, the Consumer and RBAC user are
+  also deleted, but accessing the `/workspaces/workspace_name/meta` endpoint would show counts for Consumers
+  and RBAC users, which prevented the workspace from being deleted. Now deleting entities correctly updates
+  the counts, allowing an empty workspace to be deleted.
+- Updates Kong Dev Portal templates' JQuery dependency to v3.6.0, improving security.
+- Users with the `kong_admin` role can now log in to Kong Manager when `enforce_rbac=both` is set. 
+- Renamed the property identifying control planes in hybrid mode when using Kong Vitals with anonymous
+  reports enabled. Before, users received the error, `Cannot use this function in data plane`, on their control planes.
+
+#### Plugins
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  With this fix, when the OpenID Connect plugin is configured with a `config.anonymous` consumer and
+  `config.scopes_required` is set, a token missing the required scopes will have the anonymous consumer
+  headers set when sent to the upstream.
+- Fixes a regression in the Kong Collector plugin that caused HARs to be kept in its queue indefinitely.
+
+
```

Comment:
```suggestion
#### Hybrid Mode
- Control Planes are now more lenient when checking Data Planes' compatibility in Hybrid mode. See the Version compatibility section of the Hybrid Mode guide for more information. #7488

```
 see:
- https://konghq.atlassian.net/browse/FTI-2704
- https://konghq.atlassian.net/browse/FT-1831
 
 The backport + the removal of extra fields was done to ensure that 2.4.1.2 could communicate with older dataplanes <2.3.3.3 just like 2.5.0.0; I don't know if we want to call out that FTI directly or even the wording (@eskibars any thoughts).

Reply:
will add the release note I wrote for those updates with links

Reply:
oh :hankey: ... I didn't add anything to the changelogs on confluence for these. I will update their too.

Reply:
+1 to not having FTIs in our changelogs